src/main/scala/tests/genericMethods.scala

```diff
@@ -9,8 +9,8 @@ class Types:
     def mixedAndAndOr(base: Int & String, nested: List[Int | Double] & Seq[String]): Unit 
     = ???
 
-    def literal(i: 1, d: 3.3, c: 'a'): 34 
-    = 34
+    def literal(i: 1, d: 3.3, c: Char): 34 
+    = 34 /* TODO: Deleted default value to repair searchbar until fix is merged */
```

Comment:
This is not default value but type signature (so `c` is always `'a'`).

Please create a task here and link its number using #taskNumer

Reply:
Created